osquery/tables/system/posix/sudoers.cpp

```diff
@@ -30,6 +30,8 @@ const std::string kSudoFile = "/etc/sudoers";
 const std::string kSudoFile = "/usr/local/etc/sudoers";
 #endif
 
+const char* kSudoWhitespaceChars = "\t\v ";
```

Comment:
```suggestion
const std::string kSudoWhitespaceChars = "\t\v ";
```

can this be `const std::string` for consistency (we use `std::string` for most other consts).